pkg/webhook/pod.go

```diff
@@ -859,13 +859,7 @@ func getBaseSecurityContext(podSecurityContext *corev1.PodSecurityContext, vault
 		AllowPrivilegeEscalation: &vaultConfig.PspAllowPrivilegeEscalation,
 		ReadOnlyRootFilesystem:   &vaultConfig.ReadOnlyRootFilesystem,
 		Capabilities: &corev1.Capabilities{
-			Add: []corev1.Capability{
-				"CHOWN",
-				"SETFCAP",
-				"SETGID",
-				"SETPCAP",
-				"SETUID",
-			},
+			Add: []corev1.Capability{},
```

Comment:
Hi @sedadas, sorry for the delayed response. Just wanted to note here: we don't want to fully remove these options. Rather, we can make them part of the vault config and make them configurable and optional to specify. Would that work for you?

Reply:
Hello @ramizpolic and happy new year!

Yes, that would work for me and I suppose for other users as well. The lines you mentioned are however part of the base security context, which is applied to the pods. From what I gathered by reading the issues, though, these capabilities are needed for agent templating, which can be explicitly enabled. By reading the code, I see that on line 796 the capabilities are again added to the security context when enabling agent templating, which would mean that setting them in the base security context is not needed.

May I suggest that we merge this PR (bugfix) so that users are able to update to newer versions and then we can create another PR for setting the capabilities through the CR (feature)?

Reply:
Sure, merged. We will do a release of BV tools this week so let us know if there are any issues. Thanks for the contribution @sedadas!